BUILD.ghc

```diff
@@ -43,4 +43,9 @@ haskell_toolchain(
     c2hs = "@c2hs//:bin",
     version = "8.2.2",
     tools = ":bin",
+    locale_archive = select({
```

Comment:
platform-core currently uses its own template rather than the one from Hazel, so that it can set custom compiler flags: https://github.com/FormationAI/platform-core/blob/f6de69dba0787d331d0143764a21a7cfec320a5d/BUILD.ghc#L41

Should we just make this change to that file directly?  Or do you think it makes sense to change both that and this file?

Reply:
Yeah I noticed that after I opened the PR. I'm not sure, why not change both after all.